flask_app.py

```diff
@@ -156,6 +156,28 @@ def reddit_callback():
     }
     return build_template_response(None, ERROR_PAGE, template_params)
 
+@app.route(DISCORD_CALLBACK_ROUTE)
+def discord_callback():
+    error = request.args.get('error', '')
+    if error:
+        return "Error: " + error
```

Comment:
I'm not sure I've ever seen this code path taken, but it should probably use ERROR_PAGE instead
```python
return build_template_response(None, ERROR_PAGE, template_params)
```